dbfit-java/core/src/main/java/dbfit/api/vendor/DefaultDbStoredProcedureCall.java

```diff
@@ -28,26 +28,23 @@ protected DbParameterAccessors getAccessors() {
         return accessors;
     }
 
-    public boolean isFunction() {
+    private boolean isFunction() {
```

Comment:
With those special cases I'm wondering whether `isFunction` is a good name or it should rather be something like `hasReturnValue`?